tests/test_sca.py

```diff
@@ -62,7 +62,8 @@ def test_archive_source(
 ):
     target_file, package_name = _archive_source(source_url, package_type)
     mock_git_archive.assert_called_once()
-    assert f"--remote={remote_name}" in mock_git_archive.call_args.args[0]
+    assert remote_name == mock_git_archive.call_args.args[0]
+    # TODO: What is below supposed to be asserting?
```

Comment:
@jasinner Any idea what this is supposed to check?

Reply:
Sorry for the mistake, it was probably suppose to check another argument to the function. I'll also fix it during my changes for CORGI-158.